packages/react-components/react-search/docs/MIGRATION.md

```diff
@@ -0,0 +1,88 @@
+# SearchBox Migration
```

Comment:
Can be a follow up but this information should be added to the doc site here: https://github.com/microsoft/fluentui/tree/master/apps/public-docsite-v9/src/Concepts/Migration